portal/templates/redesign/teach_new/base_registering.html

```diff
@@ -4,11 +4,17 @@
 
 {% block secondaryButtons %}
 <a class="button--menu button--menu--secondary button--menu--teacher--active">School / Club</a>
+{% if onboarding_done %}
+<a class="button--menu button--menu--secondary button--menu--enabled">Rapid Router</a>
+<a class="button--menu button--menu--secondary button--menu--enabled">Teaching Resources</a>
+{% else %}
 <a class="button--menu button--menu--secondary button--menu--disabled">Rapid Router</a>
 <a class="button--menu button--menu--secondary button--menu--disabled">Teaching Resources</a>
+{% endif %}
 {% endblock secondaryButtons %}
 
 {% block content %}
+{% if onboarding_done %}{% else %}
```

Comment:
@CelineBoudier Can we have something like `{% if not onboarding_done %}` here.  An empty `if` with only an `else` seems untidy to me.